app/views/dependencies/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% @title = params[:rubygem_id] %>
+<% @subtitle = @latest_version.try(:slug) %>
+<% if @dependencies.present? && @latest_version.indexed %>
+  <div class="l-full--l">
+    <% @dependencies.each_with_index do |dependency,idx| %>
```

Comment:
It would be nicer if we first show runtime and then development dependencies, also include this type info in the list.
https://guides.rubygems.org/patterns/#declaring-dependencies